### PR TITLE
Fixed a mistake in the FilenameUtils.concat()'s Javadoc about an absolute path.

### PR DESCRIPTION
**Pull request by yuji-k64613**
ng
/foo/a/ + ../bar     -->   foo/bar

ok
/foo/a/ + ../bar     -->   /foo/bar

### CODE REVIEW

**Comment by yuji-k64613** [comment]
The Travis CI build failed, what should I do?

**Comment by Stzx** [comment]
This should be a fake failure.

**Comment by garydgregory** [comment]
This could be a random failure and certainly should not be affected by a Javadoc changes.

@yuji-k64613 : If you want to be credited, please update `src/changes/changes.xml`, otherwise, please say if you do not here.

File: src/changes/changes.xml [hunk 1]
```diff
@@ -134,6 +134,9 @@ The <action> type attribute can be add,update,fix,remove.
       <action issue="IO-625" dev="ggregory" type="fix" due-to="Mikko Maunu">
         Corrected misleading exception message for FileUtils.copyDirectoryToDirectory.
       </action>
+      <action issue="IO-626" dev="Yuji Konishi" type="fix" due-to="Yuji Konishi">
```
Comment (Stzx) [inline src/changes/changes.xml]: Normally the **dev** is a committer, and **due-to** is the PR contributor name/alias.

Comment (yuji-k64613) [reply src/changes/changes.xml]: Sorry.

I thought that a committer and the PR contributor are me.
Which is my mistake, a committer or the PR contributor?

**Pull request merged**